gui/console.cpp

```diff
@@ -217,12 +218,14 @@ void ConsoleDialog::drawLine(int line) {
 
 	y += line * kConsoleLineHeight;
 
+	int baseIdx = (start + line) * kCharsPerLine;
+	assert(baseIdx >= 0 && baseIdx + limit <= kBufferSize);
```

Comment:
There's no reason to add an assert in this case. Just return if we are trying to access invalid memory.

Reply:
Gotcha. I'm hesitant to add silent failure so I'll drop that commit. (I'm just a beginner, though.) Someone else may add that instead.